pyvrp/cpp/Solution.h

```diff
@@ -149,12 +150,13 @@ class Solution
 private:
     using Routes = std::vector<Route>;
 
-    size_t numClients_ = 0;       // Number of clients in the solution
-    Distance distance_ = 0;       // Total distance
-    Load excessLoad_ = 0;         // Total excess load over all routes
-    Cost prizes_ = 0;             // Total collected prize value
-    Cost uncollectedPrizes_ = 0;  // Total uncollected prize value
-    Duration timeWarp_ = 0;       // Total time warp over all routes
+    size_t numClients_ = 0;         // Number of clients in the solution
+    Distance distance_ = 0;         // Total distance
+    Load excessLoad_ = 0;           // Total excess load over all routes
+    Cost prizes_ = 0;               // Total collected prize value
+    Cost uncollectedPrizes_ = 0;    // Total uncollected prize value
+    Duration timeWarp_ = 0;         // Total time warp over all routes
+    size_t numMissingClients_ = 0;  // Number of required but missing clients
```

Comment:
Have you considered 'unplanned' rather than 'missing'? (with a slight preference for unplanned I'm fine with either)

Reply:
We do use unplanned in another context (greedy repair for crossover), but I feel like that's not the same thing as missing. The unplanned customers are just all clients that are not in the current solution, but they need not be missing if they're not required. I think those that are missing are unplanned _and_ required.